deploy/charts/cert-manager/templates/rbac/controller-certificates.yaml

```diff
@@ -23,9 +23,11 @@ rules:
   - apiGroups: ["cert-manager.io"]
     resources: ["certificates/finalizers", "certificaterequests/finalizers"]
     verbs: ["update"]
+{{- if not (has "orders" .Values.disableControllers) }}
```

Comment:
Orders are being watched in certificate-requests controller, so we need to keep the RBAC to list and watch orders https://github.com/cert-manager/cert-manager/blob/master/pkg/controller/certificaterequests/acme/acme.go#L78